lib/cartopy/mpl/ticker.py

```diff
@@ -233,18 +373,58 @@ def __init__(self,
             lon_formatter = LongitudeFormatter()
             ax.xaxis.set_major_formatter(lon_formatter)
 
+
+        When not bounded to an axis::
```

Comment:
Same again please, a bit more detail about what this does, and `bounded` -> `bound`.

Reply:
Fixed